packages/react-sdk/docusaurus/docs/React/01-basics/04-installation.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: 04-installation
+title: Installation
+---
+
+The installation of the packages for the Stream Video React SDK can be either done with `yarn` or `npm`.
+
+### Install the React SDK
+
+For `yarn`:
+
+```
+yarn add @stream-io/video-react-sdk
+```
+
+For `npm`:
+
+```
+npm install @stream-io/video-react-sdk
+```
+
+### Install the low-level client
```

Comment:
I think we should mention the `react-bindings` here. They are implicit within the SDK but, if one wants to build their app on top of our LLC, they can come quite handy!

- `yarn add @stream-io/video-react-bindings` and `npm install @stream-io/video-react-bindings`

Reply:
Added, with a quick explanation.